src/app/helpers/isAmpPath.test.js

```diff
@@ -0,0 +1,13 @@
+import isAmpPath from './isAmpPath';
+
+describe('isAmpPath', () => {
+  it('should return false when passed a non-article path', () => {
+    expect(isAmpPath('/random')).toEqual(false);
+  });
+  it('should return false when passed a non-AMP article path', () => {
+    expect(isAmpPath('/news/articles/c0000000001o')).toEqual(false);
+  });
+  it('should return true when passed an AMP article path', () => {
+    expect(isAmpPath('/news/articles/amp/c0000000001o')).toEqual(false);
```

Comment:
This is asserting that the AMP regex path helper returns `false`, which doesn't seem right?